competencies/roles/TechLeaderLevelMapping.md

```diff
@@ -0,0 +1,7 @@
+                                                                                                                                                                                                
+| Title/Job                             |  Industry Comparison       | Key Competencies                                                                                | Other Competencies              |     Notes  |
+| ------------------------------------- |:-------------------------- |:-----------------------------------------------------------------------------------------------:| -------------------------------:| ----------:|
+| Team Lead                             |  L2 first non-IC role      | Proficient: "Releasing Value"  and "Communication"                                              | 50%  proficient or above prof   |  Lateral to SWDev IV |
+| Eng Manager                           |  L3 typically 2 - 5 years  | Proficient: "Developing Team" and "Facilitation"                                                | 75% proficient; some authority  |  | 
+| Senior Eng Manager / Head of          |  L4 typically < 8 years    | Proficient or Authority "Communication" and "Tech Strategy, Alignment, Directional Leadership"  | 50% proficient; 50% authority   |  | 
```

Comment:
Re: "industry comparison" column:
- Are those comparison levels (L2, L3, L4) a reference to our defined levels for other roles, or are they just illustrative/examples? I fear they might be more confusing than clarifying because level definitions vary so much industry-wide (e.g. I have never seen "team lead" associated with "L2" in other companies).
- I assume the "years of experience" in that column is meant to be "years of experience in tech leadership role". The "< 8 years" one read odd to me because of the "less than" - should it be e.g. "typically > 5 years" instead?

Reply:
There is an industry resource we use that has a range of 5+ to <8 and I was trying to lean to the top of that range but yeah a minimum makes more sense. 

The L2 for team lead is a good flag, will try to rejig

Reply:
Updated to try to communicate that Team lead typically starts after a couple levels of IC experience in the same domain (e.g. junior development role, intermediate development role, then either senior development role or team lead role - senior IC and Team Lead are peer roles in this model )